docs/usage.rst

```diff
@@ -95,11 +95,12 @@ optional specifications:
 
 The :meth:`pystac_client.Client.conforms_to` method is used to check conformance
 against conformance classes (specs). To check an API for support for a given spec,
-pass the `conforms_to` function the name of a :class:`ConformanceClasses`.
+pass the `conforms_to` function a :class:`ConformanceClasses` enum value.
```

Comment:
The str approach is the intended functionality. Do you think using the enum explicitly is better for users?

Reply:
I thought the intention was to use the enum, and then string was just an alternate, since using the enums reduces the risk of accidental misspellings

Reply:
I think we intended data consumers to use at str so they don't have to do the extra import, pystac internally to use the enum.

Reply:
I don't like that, but I also didn't intend to start editing the usage.rst file anyway, so reverting.